pulsar-functions/instance/src/main/python/log.py

```diff
@@ -34,6 +34,19 @@
 # see time formatter documentation for more
 date_format = "%Y-%m-%d %H:%M:%S %z"
 
+class LogTopicHandler(logging.Handler):
+  def __init__(self, topic_name, pulsar_client):
+    Log.info("Setting up producer for log topic %s" % topic_name)
+    self.producer = pulsar_client.create_producer(
+      str(topic_name),
+      block_if_queue_full=True,
+      batching_enabled=True,
+      batching_max_publish_delay_ms=1,
```

Comment:
We could use bigger timeout `10ms` or even `100ms`, since this is not latency sensitive, to improve efficiency.

Reply:
Also we should enable LZ4 compression by default. It's very light on cpu and provides good compression for basic text